src/content/docs/en/reference/programmatic-reference.mdx

```diff
@@ -246,4 +277,94 @@ import { sync } from "astro";
 await sync({
   root: "./my-project",
 });
-```
\ No newline at end of file
+```
+
+## `mergeConfig()`
+
+<p>
+
+**Type:** `<T extends AstroConfig | AstroInlineConfig>(config: T, overrides: DeepPartial<T>) => T`
+</p>
+
+Imported from `astro/config`, merges a partial Astro configuration on top of an existing, valid, Astro configuration.
+
+The return value is a valid Astro config combining both values such that:
```

Comment:
```suggestion
`mergeConfig()` accepts two Astro config objects, and the return value is a valid Astro config combining both values such that:
```
Is this correct to say? At first, I didn't realize you needed to give it two different configs to merge; I thought it was "merge whatever you already have with this new thing I'm giving you", so it's maybe helpful to say this out in words (not just in the type declaration?)

Reply:
The first argument is a config, the second is any part of a config, everything is optional on it (that is the `DeepPartial` in the types). Sending another option!